kombu_fernet/serializers/__init__.py

```diff
@@ -42,3 +42,5 @@ def force_text(func):
     def inner(message):
         if isinstance(message, six.binary_type):
             message = message.decode('utf-8')
+        return func(message)
+    return inner
```

Comment:
The decorator was incomplete.
[Here](https://github.com/BetterWorks/kombu-fernet-serializers/blob/master/kombu_fernet/serializers/json.py#L12) is called and should return a function, [this](https://github.com/BetterWorks/kombu-fernet-serializers/blob/master/tests/test_serializers.py#L17) test will fail without the modifications.
I saw the modifications also on the [heroku](https://github.com/heroku/kombu-fernet-serializers/blob/master/kombu_fernet/serializers/__init__.py#L37) repo also.